lib/client.ts

```diff
@@ -115,8 +115,19 @@ export class Client {
       () => notNil(params.id),
       collectionUrl => `${collectionUrl}/${params.id}`
     ),
-    (hostname) => `${hostname}${S(base).template(params, '{', '}').s}`
+    (hostname) => `${hostname}/${this.version(api_version)}${S(base).template(params, '{', '}').s}`
   )(this.host)
+
+  private version = (api_version: string): string => {
```

Comment:
It would be better to refactor it with `Ramda` FP style. I just little bit stuck with that.